crossbeam-epoch/src/sync/list.rs

```diff
@@ -247,32 +247,39 @@ impl<'g, T: 'g, C: IsElement<T>> Iterator for Iter<'g, T, C> {
                 // node leaves the list in an invalid state.
```

Comment:
Shouldn't this comment actually say "The tag should always be zero, .."?

Reply:
Thanks!  just fixed it.